apps/site/src/views/Resources/interfaces/interfaces.ts

```diff
@@ -0,0 +1,24 @@
+export interface tag {
```

Comment:
Nitpick: Since `tag` is used to type component props, it should have "Props" at the end like you have with the other interfaces.

Reply:
it is nested tho, so I figured it would be fine as it is.